client/jetpack-cloud/sections/onboarding-tours/constants.ts

```diff
@@ -4,4 +4,5 @@
 export const JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME: Record< string, string > = {
 	addSiteStep1: 'jetpack-cloud-site-dashboard-add-new-site-tour-step-1',
 	addSiteStep2: 'jetpack-cloud-site-dashboard-add-new-site-tour-step-2',
+	dashboardWalkthrough: 'jetpack-cloud-sites-overview-dashboard-walkthrough-tour',
```

Comment:
Just realised the prefix is `jetpack-cloud-sites` and the other preferences are `jetpack-cloud-site`, which don't quite match.

Any reason we didn't go with `jetpack-manage`?

`jetpack-manage-add-new-site-tour-step-1`
`jetpack-manage-add-new-site-tour-step-2`
`jetpack-manage-dashboard-walkthrough-tour` <--though this name is misleading

Reply:
No reason at all, will change everything!